codegen-lite-maven-plugin/pom.xml

```diff
@@ -74,10 +74,6 @@
             <artifactId>maven-project</artifactId>
             <groupId>org.apache.maven</groupId>
         </dependency>
-        <dependency>
```

Comment:
Why did we need to remove these?

Reply:
It started to fail because they were "unused declared dependencies"

Reply:
gotcha

Reply:
Turns out there was a typo in the pom file, fixed it and reverted the change